frontend/src/components/Pages/ItemList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Item, State } from "../../constants/Interfaces";
+import { View } from "react-native";
+import { ItemTag, ItemPrice } from '../Item';
+import { PageStates } from '../../constants/States';
+
+interface Props {
+    items: Item[],
+    currState: PageStates
+}
+
+// item list for items user selected
+export default function ItemList(props: Props) {
+    let ouputedItems = [];
+
+    for (const item of props.items) {
+        if (props.currState == PageStates.itemListPrices) // item display with price and store
```

Comment:
```suggestion
        if (props.currState === PageStates.itemListPrices) // item display with price and store
```
It's a good habit to use strict equality in JS, as it's what you want most of the time